sentinel-core/src/main/java/com/alibaba/csp/sentinel/config/SentinelConfig.java

```diff
@@ -61,20 +61,25 @@
     public static final String STATISTIC_MAX_RT = "csp.sentinel.statistic.max.rt";
     public static final String SPI_CLASSLOADER = "csp.sentinel.spi.classloader";
     public static final String METRIC_FLUSH_INTERVAL = "csp.sentinel.metric.flush.interval";
+    public static final String MAX_CONTEXT_NAME_SIZE = "csp.sentinel.context.max.size";
 
     public static final String DEFAULT_CHARSET = "UTF-8";
     public static final long DEFAULT_SINGLE_METRIC_FILE_SIZE = 1024 * 1024 * 50;
     public static final int DEFAULT_TOTAL_METRIC_FILE_COUNT = 6;
     public static final int DEFAULT_COLD_FACTOR = 3;
     public static final int DEFAULT_STATISTIC_MAX_RT = 5000;
     public static final long DEFAULT_METRIC_FLUSH_INTERVAL = 1L;
+    public static final int DEFAULT_MAX_CONTEXT_NAME_SIZE = 2000;
+
+    private static int maxContextNameSize = DEFAULT_MAX_CONTEXT_NAME_SIZE;
 
```

Comment:
I think we should name it `maxContextSize`

Reply:
It named MAX_CONTEXT_NAME_SIZE in com.alibaba.csp.sentinel.Constants